pkgs/development/libraries/retro-gtk/default.nix

```diff
@@ -0,0 +1,57 @@
+{ lib
+, stdenv
+, fetchurl
+, cmake
+, meson
+, ninja
+, pkg-config
+, epoxy
+, glib
+, gtk3
+, libpulseaudio
+, libsamplerate
+, gobject-introspection
+, vala
+, gtk-doc
+}:
+
+stdenv.mkDerivation rec {
+  pname = "retro-gtk";
+  version = "1.0.2";
+
+  src = fetchurl {
+    url = "mirror://gnome/sources/retro-gtk/${lib.versions.majorMinor version}/${pname}-${version}.tar.xz";
+    sha256 = "1lnb7dwcj3lrrvdzd85dxwrlid28xf4qdbrgfjyg1wn1z6sv063i";
+  };
+
+  nativeBuildInputs = [
+    meson cmake pkg-config epoxy glib gtk3 libpulseaudio libsamplerate gobject-introspection vala gtk-doc ninja
```

Comment:
Build tools should go to `nativeBuildInputs`. Runtime dependencies to `buildInputs`.

https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/build-tools-in-build-inputs.md

Reply:
Fixed!